scanner/pawn.lua

```diff
@@ -73,6 +74,63 @@ scans.baseMaxHealth = inheritClass(Scan, {
 	end
 })
 
+scans.bonusMove = inheritClass(Scan, {
+    id = "BonusMove",
```

Comment:
The whitespace (spaces/tabs) used in this function is quite different from the rest of memedit's code.

memedit uses tab for indentation, and no tab at empty lines.

Reply:
Yup, that's my bad, a copy paste error on my part.